setup.cfg

```diff
@@ -43,6 +43,7 @@ project_urls =
 [options]
 packages = find:
 install_requires =
+    dill>=0.3.4
```

Comment:
Does it need to be version `0.3.4` or would it work with an older version, too?
If we can, we should try to keep the requirements as open as possible.

Reply:
I wasn't sure about this.

I'll go back and see if this works on other versions as well.

Reply:
https://pypi.org/project/dill/#history

0.2b1 doesn't work, 0.2 does. Updated!

Reply:
👍🏻 I couldn't find a good changelog which does make it difficult to know what exactly changed. `0.2` should be broad enough though.

Reply:
I think they were missing a dependency, or at least an error popped up that from a google search seemed related to incorrect dependency issues in other projects as well.
Based on their release history `0.2` seems to be the first "full" version.

Reply:
> Based on their release history `0.2` seems to be the first "full" version.

I agree. Although a broad dependency range is better, we don't need to encourage old beta releases. No need to go there.